src/openrct2/ride/Ride.h

```diff
@@ -464,9 +464,11 @@ enum
     RIDE_ENTRY_FLAG_PLAY_DEPART_SOUND = 1 << 3,
     RIDE_ENTRY_FLAG_ALTERNATIVE_SWING_MODE_1 = 1 << 4,
     // Twist type rotation ride
-    RIDE_ENTRY_FLAG_ALTERNATIVE_ROTATION_MODE_1 = 1 << 5,
+    RIDE_ENTRY_FLAG_ALTERNATIVE_ROTATION_MODE_1_DEPRECATED = 1 << 5,
     // Lifting arm rotation ride (enterprise)
-    RIDE_ENTRY_FLAG_ALTERNATIVE_ROTATION_MODE_2 = 1 << 6,
+    RIDE_ENTRY_FLAG_ALTERNATIVE_ROTATION_MODE_2_DEPRECATED = 1 << 6,
```

Comment:
Do we have a reason for keeping those deprecated enumerators?

I know we've done it before, looking a few lines below, but I don't remember if it was discussed or if there was any reasoning behind it, given they're not used anymore.